src/core/qgsvectorlayer.cpp

```diff
@@ -696,15 +696,12 @@ long QgsVectorLayer::featureCount( const QString &legendKey ) const
   return mSymbolFeatureCountMap.value( legendKey );
 }
 
-
-
 QgsVectorLayerFeatureCounter *QgsVectorLayer::countSymbolFeatures()
 {
+
   if ( mSymbolFeatureCounted || mFeatureCounter )
     return mFeatureCounter;
 
-  mSymbolFeatureCountMap.clear();
```

Comment:
Changes in this file which are still present very much look like leftovers

Reply:
yes, should be back to its original state now